config/foreman-answers.yaml

```diff
@@ -11,6 +11,17 @@
 ---
 foreman:
   configure_epel_repo: false
+  server_ssl_crl: ""
+  server_ssl_ca: "/etc/foreman-pki/certs/ca/ca.crt"
```

Comment:
Just to be explicit about this: I hope we're not ending up with passing this via answers and this is just so you can iterate quickly.

Reply:
This is to help with iterating. Is this asking that in the end these paths are the defaults in the puppet modules?

Reply:
Yes

Reply:
Definitely, assuming we arrived at an agreed upon file layout I planned to do that. You can see some of my testing code that did this here: https://github.com/theforeman/puppet-foreman/pull/856/files